src/scopes/activityReport/programType.js

```diff
@@ -0,0 +1,16 @@
+import { Op } from 'sequelize';
+import { filterAssociation } from './utils';
+
+const programTypes = `
+    SELECT "ActivityRecipients"."activityReportId" FROM "ActivityRecipients"
+    INNER JOIN "Grants" ON "Grants".id = "ActivityRecipients"."grantId"
+    INNER JOIN "Programs" ON "Programs"."grantId" = "Grants"."id" 
+    WHERE "Programs"."programType"`;
```

Comment:
What is the goal in the where, are you filtering out null values?

Reply:
I did add some comments, although maybe a refactor would also be in order, not sure